storage/test/storage/SimpleArrayStorageTest.java

```diff
@@ -0,0 +1,10 @@
+package storage;
+
+import storage.simple.SimpleArrayStorage;
+
+public class SimpleArrayStorageTest extends ArrayStorageTest {
+
+    public SimpleArrayStorageTest() {
+        super(new SimpleArrayStorage(MAX_TEST_STORAGE_SIZE));
```

Comment:
There is no need in modifying the original storage size. There is no business case for such thing.

Reply:
Done